apps/src/sharedFunctionalBlocks.js

```diff
@@ -402,13 +421,17 @@ function installCond(blockly, generator) {
         .appendTitle(new Blockly.FieldLabel('else', options));
       var defaultInput = this.appendFunctionalInput('DEFAULT')
         .setInline(true);
-      defaultInput.setHSV.apply(defaultInput, Blockly.FunctionalTypeColors.None);
+      defaultInput.setHSV.apply(defaultInput, Blockly.FunctionalTypeColors[type]);
 
       this.appendDummyInput('PLUS')
         .appendTitle(plusField)
         .setInline(true);
 
-      this.setFunctionalOutput(true);
+      if (type !== Blockly.BlockValueType.NONE) {
```

Comment:
Should this operator be flipped? i.e., if there _is_ `===` no type, omit the type check